### ISSUE TITLE
11. Privacy Considerations, level of assurance

### ISSUE THREAD

**Issue by mcr** [ietf-rats-wg/architecture#217]
````
Some Attesters might want a stronger level of assurance
   of the trustworthiness of a Verifier before sending Evidence to it.
   In such cases, an Attester can first act as a Relying Party and ask
   for the Verifier's own Attestation Result, and appraising it just as
   a Relying Party would appraise an Attestation Result for any other
   purpose.
````

Is this a description of mutual authentication?  If so, should it use the term?
In this case, how would each side decide what to reveal if it has to attest the other side first?
Seems like, in practice, there’d be some kind of bootstrapping ladder of trust?  i.e., they have to agree on each other’s identity before moving on to attestation results?

You could introduce the TCG Privacy CA idea as a way to solve this deadlock, but I’ll admit I don’t think that approach has many fans these days.

**Comment by nedmsmith** [comment]
Concur that it appears to be describing 'mutual-attestation'. We should define the concept then use it to describe the privacy consideration. 

The privacy consideration in this paragraph appears to be:
"Evidence may contain privacy sensitive information" 
"Attester is responsible for evaluating whether disclosure of Evidence to a Verifier is safe (from a privacy perspective)"

It is an aside IMO whether or not mutual attestation addresses sufficiently the privacy consequences of disclosure of Evidence. The Verifier might be highly domain isolated and authenticated, but still not behave in a way that protects the privacy interests of the Attester.

**Comment by mcr** [comment]
fixed in 491676868a8105e829410d4f70f9722915ff5daf
